drizzlepac/runastrodriz.py

```diff
@@ -813,12 +815,14 @@ def verify_alignment(inlist, calfiles, calfiles_flc, trlfile,
                     else:
                         trlstr = "Could not align {} to absolute astrometric frame\n"
                         trlmsg += trlstr.format(row['imageName'])
+                        print(trlmsg)
```

Comment:
Should this be _updateTrlFile(trlfile, trlmsg) and no print statement?

Reply:
A call to _updateTrlFile( ) would actually be helpful here (see 7a46930) to insure that these comments actually make it to the trailer file as a 'permanent' record of the processing.  However, this information does not make it to stdout for the user or processing system to see unless we add the 'print()' statement, where it can be logged by the pipeline separately.  These are both necessary here since the next line causes the code to exit early before adding this information to any trailer file or stdout.